### PR TITLE
[ActionButton] Add styles for labels.

### PR DESCRIPTION
**Pull request by NewOldMax**
See #75

### CODE REVIEW

File: src/styles/getTheme.js [hunk 1]
```diff
@@ -82,6 +82,9 @@ export default function getTheme(theme, ...more) {
                 borderRadius: (spacing.actionButtonSize - 16) / 2,
                 backgroundColor: grey500,
             },
+            speedDialActionLabel: {
+                color: grey600,
```
Comment (xotahal) [inline src/styles/getTheme.js]: Hey @NewOldMax :) We should probably use `palette.secondaryTextColor`. What do you think?

Comment (NewOldMax) [inline src/styles/getTheme.js]: You right. I will fix at monday.

**Comment by NewOldMax** [comment]
As I see, test errors don't related to this PR

**Comment by xotahal** [comment]
True, I fixed them [here](https://github.com/xotahal/react-native-material-ui/commit/3d8a86b0df5b3d4990ea8b1094acec3dc8d3d48d). Could you please update your fork to include that commit. I don't want to push those error to master again ...

**Pull request merged**